x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/index.ts

```diff
@@ -299,9 +301,94 @@ export class APMEventClient {
 
     return this.callAsyncWithDebug({
       operationName,
-      requestType: 'terms_enum',
+      requestType: '_terms_enum',
       params: requestParams,
       cb: (opts) => this.esClient.termsEnum(requestParams, opts),
     });
   }
+
+  async indexTemplate(
+    operationName: string,
+    params: IndicesGetIndexTemplateRequest
+  ) {
+    return this.callAsyncWithDebug({
+      operationName,
+      requestType: '_index_template',
+      params,
+      cb: async (abortOptions) => {
+        try {
+          return await this.esClient.indices.getIndexTemplate(
+            params,
+            abortOptions
+          );
+        } catch (e) {
+          if (e instanceof errors.ResponseError && e.statusCode === 404) {
+            return {
+              body: e.body,
+            };
+          }
+
+          throw e;
+        }
+      },
+    });
+  }
+
+  async simulateIndexTemplate(
+    operationName: string,
+    params: {
+      index_patterns: string[];
+    }
+  ) {
+    return this.callAsyncWithDebug({
+      operationName,
+      requestType: '/_index_template/_simulate',
+      params,
+      cb: (abortOptions) => {
+        return this.esClient.transport.request<IndicesSimulateTemplateResponse>(
+          {
+            method: 'POST',
+            path: '/_index_template/_simulate',
+            body: params,
+          },
+          abortOptions
+        );
+      },
+    });
+  }
+
+  async dataStreams(
+    operationName: string,
+    params: IndicesGetDataStreamRequest
+  ) {
+    return this.callAsyncWithDebug({
+      operationName,
+      requestType: 'data_streams',
```

Comment:
```suggestion
      requestType: '_data_streams',
```
to be consistent with the other request types. Unless it was on purpose

Reply:
Good catch. It was not on purpose. That being said, I don't think we really use `requestType`. We way want to reconsider having it.